### PR TITLE
Feature/meson ci job

### PR DESCRIPTION
**Pull request by lumurillo**
Add Meson job to the CI.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -41,9 +41,55 @@ jobs:
...
+    - name: Build r2i
+      run: |
+        NOCONFIGURE=1 ./autogen.sh
+        ./configure --disable-docs --enable-tensorflow
+        make
+        make install
+    - name: Checkout gst-inference repo
+      uses: actions/checkout@v2
+      with:
+        remote: true
```
Comment (c-trejos) [inline .github/workflows/main.yml:81]: Can we verify is this is still necessary?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -41,9 +41,55 @@ jobs:
...
+        ./configure  --disable-docs --prefix $PREFIX --libdir $PREFIX
         make
     - name: Check gst-inference
       env:
         LD_LIBRARY_PATH: /usr/local/lib/
       run: make check VERBOSE=1
+    - name: Install gst-inference
+      run: |
+        make install
+        ldconfig
```
Comment (c-trejos) [inline .github/workflows/main.yml:53]: Is this necessary, or can it be replaced by setting an env variable like we did for r2i?

Comment (lumurillo) [reply .github/workflows/main.yml]: I removed the `ldconfig` and set the env variable `LD_LIBRARY_PATH: /usr/local/lib/`

File: .github/workflows/main.yml (hunk 1 above)
Comment (lumurillo) [reply .github/workflows/main.yml]: No needed, so I removed it from both jobs

**Pull request merged**